opentelemetry-api/tests/attributes/test_attributes.py

```diff
@@ -180,3 +180,29 @@ def test_immutable(self):
         bdict = BoundedAttributes()
         with self.assertRaises(TypeError):
             bdict["should-not-work"] = "dict immutable"
+
+    # Supporting test case for a commit titled: Fix class BoundedAttributes to have RLock rather than Lock
```

Comment:
Reference to the issue are more future proof, while at it could you please move the comment as the test method docstring?

Reply:
Agreed

Reply:
Thanks, I meant to substitute the first phrase with a "See #3858" at the end of the comment

Reply:
Sorry, do you mean changing the first line of the docstring "Supporting test case for a commit titled: Fix class BoundedAttributes to have RLock rather than Lock" to "Supporting test case for a commit titled: Fix class BoundedAttributes to have RLock rather than Lock. See #3858"?

Reply:
I'm not sure whether this is the way you said - could you please check the latest commit?